GeoIP2/Http/SyncClient.cs

```diff
@@ -22,21 +24,29 @@ ProductInfoHeaderValue userAgent
             )
         {
             _auth = auth;
+#if !NETSTANDARD1_4
             _timeout = timeout;
             _userAgent = userAgent.ToString();
+#endif
         }
 
         public Response Get(Uri uri)
         {
             var request = (HttpWebRequest) WebRequest.Create(uri);
+#if !NETSTANDARD1_4
             request.Timeout = _timeout;
             request.UserAgent = _userAgent;
+#endif
             request.Headers["Authorization"] = $"Basic {_auth}";
 
             HttpWebResponse response;
             try
             {
+#if !NETSTANDARD1_4
                 response = (HttpWebResponse) request.GetResponse();
+#else
+                response = (HttpWebResponse) request.GetResponseAsync().Result;
```

Comment:
Good catch. I have changed it to `GetAwaiter().GetResult()` to avoid blocking the consumer thread. Now it will throw the same exceptions instead of wrapping them in `AggregatedException`. :)